vllm_gaudi/extension/bucketing/common.py

```diff
@@ -120,6 +132,13 @@ def generate_prompt_buckets(self):
                                                    self.max_num_seqs, self.max_num_prefill_seqs,
                                                    self.max_num_batched_tokens, self.block_size, self.num_hpu_blocks)
             self.log_generate_info(True)
+            if self.use_sliding_window:
+                self.prompt_buckets = [
+                    t for t in self.prompt_buckets
+                    if t[2] != 0 or (t[2] == 0 and (t[1] < self.slice_thld or
+                                                    (t[1] >= self.slice_thld and t[1] % self.slice_size == 0)))
+                ]
```

Comment:
```suggestion
                thld, size = self.slice_thld, self.slice_size
                self.prompt_buckets = [
                    t for t in self.prompt_buckets
                    if t[2] != 0 or t[1] < thld or t[1] % size == 0
                ]
```

minor, easier to read, identical outcome